retroshare-gui/src/main.cpp

```diff
@@ -19,6 +19,10 @@
  *  Boston, MA  02110-1301, USA.
  ****************************************************************/
 
+#include "util/stacktrace.h"
+
+CrashStackTrace gCrashStackTrace;
```

Comment:
what happens when running with gdb? Does gdb catch the SIGSEGV, or does it see the problem terminate normally? In the second case, it would make sense to put a #ifndef DEBUG around this to allow people to debug.

Reply:
I think gdb will probably get the SIGSEGV anyway, because if I remember well RetroShare already catch SIGPIPE, but we need to filter it out anyway when using gdb